.env.template

```
@@ -12,4 +12,6 @@ APPWRITE_IMAGES_BUCKET_ID=imagesBucket
 APPWRITE_MODELS_BUCKET_ID=modelsBucket
 APPWRITE_PROJECT_ID=
 APPWRITE_PASSWORD_FUNCTION_URL=
-APPWRITE_COMMUNITIES_COLLECTION_ID=
\ No newline at end of file
+APPWRITE_COMMUNITIES_COLLECTION_ID=
+DeepSeek_Api_BaseUr=
```

dotenv-linter (3.3.0)

[warning] [LowercaseKey] The DeepSeek_Api_BaseUr key should be in uppercase

(LowercaseKey)